### ISSUE TITLE
How to suppress blank alerts and/or persistant notifications

### ISSUE THREAD

**Issue by illigtr** [jazzyisj/unavailable-entities-sensor#55]
I often get a persistent notification that is blank....no devices are entities are listed. How can I suppress these blank notifications? I also created an alerts to my gmail integration using the same script and I also get blank alerts.

I am basically using your example yamls, verbatim.

**Comment by jazzyisj** [comment]
The next update I post will include this change.  

```
      - platform: state
        entity_id: sensor.unavailable_entities
        attribute: entity_id
        to: ~
        for: 1
```

https://github.com/jazzyisj/unavailable-entities-sensor/issues/50

**Comment by jazzyisj** [comment]
Fixed in v2.0
